test/components/Provider.spec.js

```diff
@@ -1,47 +1,45 @@
 /*eslint-disable react/prop-types*/
 
 import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import semver from 'semver'
 import { createStore } from 'redux'
-import { Provider, createProvider, connect } from '../../src/index.js'
+import { Provider, connect } from '../../src/index.js'
+import Context from '../../src/components/Context.js'
 import * as rtl from 'react-testing-library'
 import 'jest-dom/extend-expect'
 
-const createExampleTextReducer = () => (state = "example text") => state;
+const Consumer = Context.Consumer
 
 describe('React', () => {
   describe('Provider', () => {
     afterEach(() => rtl.cleanup())
+
     const createChild = (storeKey = 'store') => {
       class Child extends Component {
         render() {
-          const store = this.context[storeKey];
-          
-          let text = '';
-          
-          if(store) {
-            text = store.getState().toString()
-          }
-          
           return (
-            <div data-testid="store">
-              {storeKey} - {text}
-            </div>
+            <Consumer>
+              {(value) => {
+                return (
+                  <div>
+                    <div data-testid="store">
+                      {storeKey} - {value && value.store ? value.store.getState().mine : ''}
```

Comment:
Generally not keen on this `.mine` thing in the tests.  Will probably overwrite the test files with the versions from #1000 .

Reply:
Yeah, I had brought this up before too: https://github.com/reduxjs/react-redux/pull/996#discussion_r209653040 I think you had a solution?